using_images/xpaas_images/eap.adoc

```diff
@@ -1,119 +1,110 @@
-in the JBoss EAP xPaaS image compared to the regular release of JBoss EAP.
-
-This topic details the differences between the JBoss EAP xPaaS image and the
-regular release of JBoss EAP, and provides instructions specific to running and
-configuring the JBoss EAP xPaaS image. Documentation for other JBoss EAP
-functionality not specific to the JBoss EAP xPaaS image can be found in the
-https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/[JBoss
-EAP documentation on the Red Hat Customer Portal].
-
-`_EAP_HOME_` in this documentation, as in the
-https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/[JBoss
-EAP documentation], is used refer to the JBoss EAP installation directory. The
-location of `_EAP_HOME_` inside a JBoss EAP xPaaS image is *_/opt/eap/_*, which
-the `*JBOSS_HOME*` environment variable is set to by default.
+== Comparison Between Product and Image
 
-== Comparing the JBoss EAP xPaaS Image to the Regular Release of JBoss EAP
+The xPaas JBoss EAP images differ from the JBoss EAP product in several ways:
 
-=== Functionality Differences for OpenShift JBoss EAP xPaaS Images
+. The image does not include the JBoss EAP Management Console used to manage xPaaS JBoss EAP images.
+. The JBoss EAP Management CLI is included in the xPaaS JBoss EAP image, but can only access the Management CLI of a container from within the pod.
+. Domain mode is not supported in the xPaaS JBoss EAP image. Instead, OpenShift manages the creation and distribution of applications in the containers.
+. The image’s default root page is disable. Deploy your own application to the root context as ROOT.war.
+. The EAP 6.4 image supports A-MQ for inter-pod and remote messaging. HornetQ is only supported for intra-pod messaging and only enabled when A-MQ is absent. The EAP 7 Beta image includes Artemis as a replacement for HornetQ.
 
-There are several major functionality differences in the OpenShift JBoss EAP xPaaS image:
+For further information about JBoss EAP functionality and features independent from the JBoss EAP image, see the  https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/[JBoss EAP documentation] on the Red Hat Customer Portal.
 
-* The JBoss EAP Management Console is not available link:#Managing-OpenShift-JBoss-EAP-xPaaS-Images[to manage OpenShift JBoss EAP xPaaS images].
-* The JBoss EAP Management CLI is only bound locally. This means that you can only access the Management CLI of a container from within the pod.
-* Domain mode is not supported. OpenShift controls the creation and distribution of applications in the containers.
-* The default root page is disabled. You may want to deploy your own application to the root context (as *_ROOT.war_*).
-* A-MQ is supported for inter-pod and remote messaging. HornetQ is only supported for intra-Pod messaging and only enabled in the absence of A-MQ.
+== Comparison Between the xPaaS JBoss EAP 6.4 and 7.0 Beta Images
```

Comment:
Same comment as above, suggested title: Comparing the xPaaS JBoss EAP 6.4 and 7.0 Beta Images